data/mods/DinoMod/monsters/dinosaur.json

```
@@ -1252,4 +1252,44 @@
     "placate_triggers": [ "MEAT" ],
     "categories": [ "DINOSAUR" ]
   }
+  {
+    "type": "MONSTER",
+    "id": "mon_Qianzhousaurus",
```

```suggestion
    "id": "mon_qianzhousaurus",
```
ID formatting is supposed to be all lowercase.